monai/deploy/operators/dicom_series_to_volume_operator.py

```diff
@@ -30,6 +33,15 @@ class DICOMSeriesToVolumeOperator(Operator):
     Channel is limited to 1 as of now, and `C` is absent in the NumPy array.
     """
 
+    def __init__(self, apply_voi_lut: bool = False, *args, **kwargs):
```

Comment:
Is VOI LUT always applied if present? Would a default of True for apply_voi_lut be better?

Reply:
In my opinion, only the modality LUT should always be used. This is in accordance with e.g. ITKReader.